### PR TITLE
deisctl 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by andyshinn**
This is a management utility for [Deis](http://deis.io/) clusters. It goes along with the recent Deis client added.

### CODE REVIEW

**Comment by phspagiari** [comment]
+1

File: Library/Formula/deisctl.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    end
+
+    cd "deisctl" do
+      system "godep", "go", "build", "-a", "-ldflags", "-s", "-o", "dist/deisctl"
+      bin.install "dist/deisctl"
+    end
+  end
+
+  test do
+    system "#{bin}/deisctl", "help"
```
Comment (phspagiari) [inline Library/Formula/deisctl.rb]: As @andyshinn said, there is no other command that can be ran without a deis platform running, so the tests cannot be more more complex than this.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**